symbolic-base/src/ZkFold/Symbolic/Data/Conditional.hs

```diff
@@ -21,23 +18,14 @@ import           ZkFold.Symbolic.Data.Combinators (mzipWithMRep)
 import           ZkFold.Symbolic.MonadCircuit     (newAssigned)
 
 -- TODO: move to ZkFold.Symbolic.Data.Bool
-
-instance (Symbolic c, LayoutFunctor f) => Conditional (Bool c) (c f) where
-    bool x y (Bool b) = restore $ \s ->
-      ( symbolic3F b (arithmetize x s) (arithmetize y s)
-          (\(Par1 c) f t -> if c Prelude.== zero then f else t)
-          \(Par1 c) -> mzipWithMRep $ \i j -> do
-            i' <- newAssigned (\w -> (one - w c) * w i)
-            j' <- newAssigned (\w -> w c * w j)
-            newAssigned (\w -> w i' + w j')
-      , let Par1 wb = witnessF b
-         in mzipWithRep
-              (\wx wy -> (one - wb) * wx + wb * wy)
-              (payload x s)
-              (payload y s)
-      )
-
-deriving newtype instance Symbolic c => Conditional (Bool c) (Bool c)
-
-instance Symbolic c => Conditional (Bool c) (Proxy c) where
-  bool _ _ _ = Proxy
+-- TODO: Can we have a conditional on symbolic functions?
```

Comment:
Yes we can, it's pointwise and it was present before refactoring of Conditional:
```haskell
bool f g b = \x -> bool (f x) (g x) b
```